### ISSUE TITLE
Fix gallery preview

### ISSUE THREAD

**Issue by CKFinderBOT** [ckfinder/ckfinder#203]
* [x] add focus styles to gallery
* [x] lock TAB inside gallery
* [x] fix button styling
* [x] Esc does not close the gallery when PDF file is viewed
* [x] Unable to cycle between files when PDF is previewed

**Comment by CKFinderBOT** [comment]
Fixed. The updated code will be publicly available in the upcoming release.
